frontend/src/app/libs/user-profile-page/store/user/user.effects.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import {
+	GetUserFromAuthAction,
+	GetUserFromAuthFailedAction,
+	GetUserFromAuthSuccessfullyAction,
+	UpdateUserAction,
+	UpdateUserInfoFailedAction,
+	UpdateUserInfoSuccessfulyAction,
+	UserActionTypes,
+} from './user.actions';
+import { Observable, of } from 'rxjs';
+import { map, exhaustMap, catchError } from 'rxjs/operators';
+import { TypedAction } from '@ngrx/store/src/models';
+import { IUser } from '../../model/user';
+import { UserService } from '../../service/user-service';
+import { AuthFacadeService } from '../../../auth/store/auth/auth.facade';
+import { EntityWrapper } from '../../../auth/models/entity-wraper';
+import { User } from '../../../auth/models/user';
+
+@Injectable()
+export class UserEffects {
+	public getUser$: Observable<{} & TypedAction<UserActionTypes>> = createEffect(() =>
+		this.actions$.pipe(
+			ofType(UserActionTypes.GetUserFromAuth),
+			// map((action: GetUserFromAuthAction) => action),
+			exhaustMap(() =>
+				this.authFacadeService.user$.pipe(
```

Comment:
you can use `.userValue$` to avoid mapping

Reply:
Dima Efremenko facade does not have such method